docker/dev/centos/README.md

```diff
@@ -0,0 +1,46 @@
+# How to use this Dockerfile
+
+This image is intended for development of Orion Context Broker. This will create an environment on which Orion can compile and be run based on CentOS 6.
+
+If you are only interested in running Orion Context Broker in a container this image is not what you are looking for. 
+
+## Build the image
+
+You only need to do this once.
+
+    docker build -t orion-dev:centos6 .
+    
+## Run the container
+
+This assumes that there is another docker running with name $mongodb$. If you don't, type this:
+
+    docker run -d --name mongodb mongo
+
+The following line will run the container exposing port 1026, give it a name -in this case $orionDev$-, link it to the mongodb docker, and present a bash prompt. Keep in mind that /path/to/orion/source is the path to your Orion Context Broker code in the host, not the container. 
```

Comment:
`/path/to/orion/source` is not actually used in the command line below... Maybe it should be used instead of `~/src/fiware-orion` ?